dataset_download/download_dataset_impl.py

```diff
@@ -25,23 +25,27 @@
 BLOCKSIZE = 65536  # for sha256 computation
 
 
+DEFAULT_DOWNLOAD_MODALITIES = [
+    "metadata",
+    # "depth_maps",   # by default we do not download depth maps!
+    "rgb_videos",
+    "mask_videos",
+    "gaussian_splats",
+    "point_clouds",
+    "sparse_point_clouds",
+    "segmented_point_clouds",
+]
+
+
 def download_dataset(
     category_to_archives_file: str,
     link_list_file: str,
     download_folder: str,
     n_download_workers: int = 4,
     n_extract_workers: int = 4,
+    download_small_subset: bool = False,
     download_super_categories: Optional[List[str]] = None,
-    download_modalities: Optional[List[str]] = [
-        "metadata",
-        # "depth_maps",   # by default we do not download depth maps!
-        "rgb_videos",
-        "mask_videos",
-        "gaussian_splats",
-        "point_clouds",
-        "sparse_point_clouds",
-        "segmented_point_clouds",
-    ],
+    download_modalities: Optional[List[str]] = DEFAULT_DOWNLOAD_MODALITIES,
```

Comment:
Probably no need to support None. This will simplify some checks below.

```suggestion
    download_modalities: List[str] = DEFAULT_DOWNLOAD_MODALITIES,
```